cloudnet/src/main/resources/lang/chinese.properties

```diff
@@ -0,0 +1,397 @@
+command-service-base-add-jvm-option-already-existing=该配置文件已具有该 JVM 选项
+command-service-base-add-jvm-option-success=JVM 选项成功添加到配置文件中
+command-service-base-add-process-parameters-option-already-existing=指定配置已经有该处理参数
+command-service-base-add-process-parameters-option-success=添加处理参数成功
+command-service-base-clear-jvm-options-success=该配置文件的 JVM 选项成功清空
+command-service-base-group-not-found=该组不存在.
+command-service-base-create-group=空组成功创建. 使用 "tasks" 命令或在配置文件中配置文件它们
+# 任务命令
+command-tasks-setup-no-console=该命令仅在控制台中可用!
+command-tasks-setup-task-already-exists=该任务已经存在!
+command-tasks-setup-question-name=新的任务名称应为?
+command-tasks-setup-question-memory=该任务的每个服务将分配到多大内存(MB)?
+command-tasks-setup-question-maintenance=默认情况下是否应为该任务激活维护? (这将防止服务自动启动)
+command-tasks-setup-question-auto-delete=停止后是否应自动从 CloudNet 中删除与该任务有关的服务? (建议: 确认(t))
+command-tasks-setup-question-static=该任务的服务应该是静态的, 是否在停止该服务后不删除数据?
+command-tasks-setup-question-startport=该任务的服务的起始端口应为?
+command-tasks-setup-question-minservices=每次自动启动该任务多少服务数?
+command-tasks-setup-question-environment=该任务的服务应使用哪个核心?
+command-tasks-setup-question-application=该任务的服务应安装哪个版本?
+command-tasks-setup-question-nodes=应在哪个 CloudNet 节点上启动该任务的服务?  (以 ";" 分隔)
+command-tasks-setup-create-begin=正在创建任务...
+command-tasks-setup-create-success=成功创建了新的任务 %name%!
+command-tasks-add-group-success=该组成功添加到任务中.
+command-tasks-add-group-already-added=该组已添加到任务中.
+command-tasks-add-node-success=任务成功分配给指定节点.
+command-tasks-add-node-already-added=该节点已添加到任务中.
+command-tasks-node-not-found=该节点不存在!
+command-tasks-remove-group-success=该组成功从任务中删除.
+command-tasks-remove-node-success=任务成功从节点中删除. 该节点上未提供所有新服务.
+command-tasks-create-task=空任务成功创建. 使用 "tasks" 命令或在配置文件中配置文件它们
+command-tasks-delete-task=任务成功删除.
+command-tasks-task-already-existing=该任务已经存在!
+command-tasks-set-property-success=你已将配置文件 %property% 从任务 %name% 更改为 %value%.
+command-tasks-reload-success=任务和组配置文件已重新加载!
+command-tasks-task-not-found=该任务不存在!
+# 组命令
+command-groups-delete-group=指定组成功删除.
+command-groups-group-already-existing=该组已存在!
+command-groups-add-environment-already-existing=该组已经在具有该环境的所有服务器上
+command-groups-add-environment-success=该环境成功添加到组中
+command-groups-remove-environment-not-found=该组没有找到指定的启动环境
+command-groups-remove-environment-success=成功删除该组服务的启动环境
+command-groups-reload-success=组的所有配置文件重新载入成功!
+# 服务命令
+command-service-service-not-found=该服务不存在
+command-service-add-template-success=成功找到模板并将其添加到等待的模板中
+command-service-add-inclusion-success=引用的内容成功添加到等待的包含中
+command-service-add-deployment-success=该部署方式已成功添加到待部署列表中
+# 模板命令
+command-template-list-versions=可用的版本类型和版本:
+command-template-list-templates=当前储存源 %storage% 的模板:
+command-template-storage-not-found=该储存源空间不存在
+command-template-invalid-version-type=该版本类型不存在
+command-template-invalid-version=该版本类型不存在该版本
+command-template-install-wrong-java=该 CloudNet 版本 %version% 与你当前的 Java 版本 %java% 不兼容, 请尝试更新 CloudNet 或前往 https://github.com/CloudNetService/CloudNet-v3/issues 进行反馈.
```

Comment:
Looks like a wrong translation here: 
```
command-template-install-wrong-java=The version %version% is not compatible with your current java version %java%
```

Reply:
These are not translation errors, but I have added content that is easy for Chinese people to understand

Reply:
solved